src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -407,7 +430,7 @@ private void UpdatePackageList(IEnumerable<PackageItemListViewModel> packages, b
                 await _list.ItemsLock.ExecuteAsync(() =>
                 {
                     // remove the loading status indicator if it's in the list
-                    Items.Remove(_loadingStatusIndicator);
+                    var removed = Items.Remove(_loadingStatusIndicator);
```

Comment:
don't use var here?

Reply:
Changed to explicit bool type declaration.